lib/players/human.rb

```diff
@@ -0,0 +1,27 @@
+require 'pry'
+module Players
+
+  class Human < Player
+
+    def move(board)
+      #retrieves input from human player
+      puts "Enter a number between 1-9:"
+      input = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression. You also don't have to check if the move is valid here. That should be done in your `turn` method. 
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```